komga/src/main/kotlin/org/gotson/komga/domain/service/SeriesLifecycle.kt

```diff
@@ -197,7 +197,7 @@ class SeriesLifecycle(
   fun getThumbnail(seriesId: String): ThumbnailSeries? {
     val selected = thumbnailsSeriesRepository.findSelectedBySeriesIdOrNull(seriesId)
 
-    if (selected != null && selected.isCustomCover) {
+    if (selected != null && selected.type == ThumbnailSeries.Type.USER_UPLOADED) {
```

Comment:
we shouldn't need this. Selected means that it's the one we want to show. When adding a custom cover by uploading, that should make it selected, so we shouldn't need this.

This would mean that even if you selected something else, you always show the user uploaded one, which is not a desired behaviour i would say.